roles/confluent.zookeeper/defaults/main.yml

```diff
@@ -24,6 +24,7 @@ zookeeper_service_overrides:
 zookeeper_service_environment_overrides:
   KAFKA_HEAP_OPTS: "-Xmx1g"
   KAFKA_OPTS: "{{ zookeeper_final_java_args | java_arg_build_out }}"
+  CONFLUENT_SECURITY_MASTER_KEY: "{% if (secrets_protection_enabled|bool) and (zookeeper_secrets_protection_config_path|length>0) %}{{hostvars['zookeeper1'].secrets_protection_masterkey}}{% endif %}"
```

Comment:
itd be better if this var was set on each host, not just zookeeper1. I have an idea that can help with this, but its tricky because we have a change in a forked repo. lets discuss